### DISCUSSION TITLE
Warning “formula separator settings conflict with locale” after update to 24.x

### DISCUSSION THREAD

**Discussion by MZeynel** [nextcloud/all-in-one#4637] [Questions] [answered]
After updating CODE server to 24.x users get the following warning each time they open a spreadsheet:
separator values warning
![image](https://github.com/nextcloud/all-in-one/assets/169546483/82d7c92d-b61c-45cd-ac89-7c9fe8afd4bd)

Only way to get rid of error is to change user language setting back to english, which isnt ideal. Ideas how to fix this issue, or at least get rid of that warning every time we open a spreadsheet?

**Comment by szaimen** [comment, marked as answer]
Hi, please open an issue about this here:https://github.com/nextcloud/richdocuments/issues
